src/pybind/mgr/dashboard/frontend/src/app/shared/services/task-manager-notification.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+
+import { ToastsManager } from 'ng2-toastr';
+import { FinishedTask } from '../models/finished-task';
+import { TaskManagerMessageService } from './task-manager-message.service';
+
+@Injectable()
+export class TaskManagerNotificationService {
+
+  constructor(public toastr: ToastsManager,
+              private taskManagerMessageService: TaskManagerMessageService) {
+  }
+
+  notify(finishedTask: FinishedTask) {
```

Comment:
Maybe this method should be moved into the new notification service, if this is merged after that PR.

Reply:
NotificationService has been merged. You can now move this method to that service.